content/wip-cards/3_card/contents.lr

```diff
@@ -0,0 +1,15 @@
+work: jfjfjf
+---
+attribution: Darth Vader
+---
+link: https://google.com/
+---
+icon_slug: fa-book
+---
+body:
+The _quick_ brown *fox* jumped `over` the lazy dog.
+> On a Saturday
+---
+languages:
+French
+Russian
```

Comment:
The only point of having this card is I wanted to see the site render with at least three cards.  We can fill this in with something real or drop it altogether.  I don't really care either way.

Reply:
* Cards adjusted to those three series
* FA icons dropped
* Unless/until I get copy for the "body", those are now blank.
* IDK the appropriate languages to tag here so those are blank now too.